dev/com.ibm.ws.jaxb_fat/publish/servers/jaxb_internal_optional_feature_toleration_fat/server.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+    Copyright (c) 2019 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+    <featureManager>
+        <feature>servlet-3.1</feature>
```

Comment:
since we don't need to install any test apps or drive any HTTP requests to the server, the `servlet-3.1` and `componenttest-1.0` features can be removed. Same comment applies to next server config also

Reply:
Features that became unnecessary after removing the app, removed from server.xml